src/main/kotlin/org/rust/ide/hints/RustParameterInfoHandler.kt

```diff
@@ -0,0 +1,159 @@
+package org.rust.ide.hints
+
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.lang.parameterInfo.*
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.PsiTreeUtil
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.util.parentOfType
+
+/**
+ * Provides funcions/methods afruments hint.
+ */
+class RustParameterInfoHandler : ParameterInfoHandler<PsiElement, RustArgumentsDescription> {
```

Comment:
There's not much of documentation on how this handler exactly works, but it doesn't show anything with `RustArgListElement`.